shared/src/persistence/elasticsearch/advancedDocumentSearch.ts

```diff
@@ -126,16 +127,21 @@ exports.advancedDocumentSearch = async ({
     ];
   }
 
-  const documentFilter = [
-    { term: { 'entityName.S': 'DocketEntry' } },
-    {
+  const documentFilter = [];
+
+  if (requireServedDate) {
```

Comment:
Is this because the pamphlets are unservable?

Reply:
Correct! Using that field as a part of the search ends up not returning any TCRPs